src/sentry/auth/helper.py

```diff
@@ -459,6 +459,27 @@ def _app_user(self) -> User | None:
     def _has_usable_password(self) -> bool:
         return bool(self._app_user and self._app_user.has_usable_password())
 
+    @cached_property
+    def _login_form(self) -> AuthenticationForm:
```

Comment:
```suggestion
    def _get_login_form(self) -> AuthenticationForm:
```

Reply:
Because this is a property, the convention is to name it as a noun, not a verb. It's being referred to below with the syntax of an attribute (`self._login_form`, not `self._login_form()`).